diff --git a/research/object_detection/protos/train.proto b/research/object_detection/protos/train.proto
index 01a188e..2c1417b 100644
--- a/research/object_detection/protos/train.proto
+++ b/research/object_detection/protos/train.proto
@@ -11,7 +11,7 @@ message TrainConfig {
   // Effective batch size to use for training.
   // For TPU (or sync SGD jobs), the batch size per core (or GPU) is going to be
   // `batch_size` / number of cores (or `batch_size` / number of GPUs).
-  optional uint32 batch_size = 1 [default=32];
+  optional uint32 batch_size = 1 [default=1];
 
   // Data augmentation options.
   repeated PreprocessingStep data_augmentation_options = 2;
